server/cmd/dcrdex/log.go

```diff
@@ -50,21 +51,29 @@ var (
 	// or data races and/or nil pointer dereferences will occur.
 	backendLog = slog.NewBackend(logWriter{})
 
+	lm = func() *dex.LoggerMaker {
+		lm, err := dex.NewLoggerMaker(backendLog, defaultLogLevel)
+		if err != nil {
+			panic(err)
+		}
+		return lm
+	}()
```

Comment:
This kind of hoop jumping is a signal that it should be done in `init`.

Reply:
Putting this initiation in `init()` will make it happen after the variables below are initialized, causing panics because the `slog.Backend` is not set yet.  I could move all the sublogger initializations as well. Unless I am mistaking something?

Reply:
Assumed setting those variables would have to go to init also, but I didn't try it out, so if it's worse, no worries.

Reply:
Actually, looks good.